.github/workflows/ci.yaml

```diff
@@ -351,6 +351,9 @@ jobs:
       - run: |
           cd mirrord/layer/tests/apps/listen_ports
           cargo build
+      - run: |
+          cd mirrord/layer/tests/apps/fork
+          gcc fork.c -o fork.test_app
```

Comment:
imo incorporate it into `./scripts` (no need to refactor, just start having better structure)

Reply:
https://github.com/metalbear-co/mirrord/pull/1591/commits/3d3f96b11b6b92240bf3d8dd9b51f087f7d05cf1
Like this?